src/Skoruba.IdentityServer4.Admin/Helpers/StartupHelpers.cs

```diff
@@ -44,7 +44,7 @@ public static void RegisterDbContexts<TContext>(this IServiceCollection services
             var storeOptions = new ConfigurationStoreOptions();
             services.AddSingleton(storeOptions);
 
-            services.AddDbContext<TContext>(options => options.UseSqlServer(configuration.GetConnectionString(ConfigurationConsts.AdminConnectionStringKey), optionsSql => optionsSql.MigrationsAssembly(migrationsAssembly)));
+            services.AddDbContext<TContext>(options => options.UseSqlServer(configuration.GetConnectionString(ConfigurationConsts.AdminConnectionStringKey), optionsSql => optionsSql.MigrationsAssembly(migrationsAssembly).UseRowNumberForPaging()));
```

Comment:
This change should be a part of separate pull request since it has nothing to do with Docker.

Reply:
I don't quite use GitHub.

Reply:
It's not a problem - we will just skip this update in the PR and you can later open a new pull request.

Rule of thumb is to add to PR only stuff related to the title of pull request, we're very happy with you contributions - this is just protip for the future :)